deploy/deploy.sh

```diff
@@ -53,13 +51,13 @@ oc apply -f $DIR/amq/address.yaml -n $NS
 
 echo "Waiting for AMQ AddressSpace to be ready"
 # unfortunately oc wait doesn't work for addressspace and address types (problem with AMQ itself)
-retry 30 5 oc get addressspace/workload-app -n $NS -o 'jsonpath={.status.isReady}' | grep 'true'
+wait_for "oc get addressspace/workload-app -n $NS -o 'jsonpath={.status.isReady}' | grep -q 'true'" "Addressspace is ready" "5m" "60"
```

Comment:
this will sleep for 60 seconds between retries? That seems a bit too long I think?

Reply:
Is it fine to make it 30s?

Reply:
probably 10 seconds should be enough, it shouldn't take that long for them to get ready. We should update them in the rest of the file too